content/docs/0.6.0/usecases/quality-gates/index.md

```diff
@@ -42,26 +42,79 @@ For more information about SLO and SLI, please take a look at [Specifications fo
 
 * **Bring your own monitored service**: This tutorial is slightly different compared to the others because you need to bring your own monitored service depending on the monitoring solution you want to use. For the sake of clarification, this tutorial uses a service called *catalogue* from the project *musicshop* meaning that you must adapt the commands to match your service and project name.  
 
+    <details><summary>*Details for a service monitored by Prometheus*</summary>
+    <p>
+    
+    This tutorial assumes that you have Prometheus monitoring configured with:
+
+    * A **scrape job** for your service. For more information about configuring a scrape job, see the official Prometheus documentation at section [scrape_config](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#scrape_config). 
+    
+        To configure a scrape job for a Prometheus deployed on Kubernetes, you need to update the `prometheus-server-conf` at the `prometheus.yml` section with an additional scrape job:
+
+            prometheus.yaml:
+            ----
+            
+            scrape_configs: 
+            - job_name: catalogue-musicshop-hardening
+              honor_timestamps: false
+              metrics_path: /prometheus
+              static_configs:
+              - targets:
+                - catalogue.musicshop-hardening:80
+
+
+    * An **alert rule** for the SLI and the scrape job. For more information about configuring alert rules, see the official Prometheus documentation at section [alerting_rules](https://prometheus.io/docs/prometheus/latest/configuration/alerting_rules/). 
+    
+        To add an alert rule to a Prometheus deployed on Kubernetes, you need to update the ConfigMap *prometheus-server-conf* at the `prometheus.rules` section with an additional group:
+
+              prometheus.rules:
+              ----
+              groups:
+              - name: catalogue musicshop-hardening alerts
+                rules:
+                - alert: response_time_p95>600
+                  expr: histogram_quantile(0.95,sum(rate(http_response_time_milliseconds_bucket{job='catalogue-musicshop-hardening'}[180s]))by(le))>600
+                  for: 5m
+                  labels:
+                    severity: webhook
+                    pod_name: catalogue
+                    service: catalogue
+                    stage: hardening
+                    project: musicshop
```

Comment:
Should we describe that these labels represent the Keptn project, service, and stage.
Therefore, they have to match.

Reply:
I added the note.